lib/sisimai/bite/email/messagelabs.rb

```diff
@@ -0,0 +1,213 @@
+module Sisimai::Bite::Email
+  module MessageLabs
+    # Sisimai::Bite::Email::MessageLabs parses a bounce email which created by
+    # Symantec.cloud: formerly MessageLabs. Methods in the module are called
+    # from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/MessageLabs.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/MAILER-DAEMON[@]messagelabs[.]com/,
+        :subject => %r/\AMail Delivery Failure/,
+      }
+      Re1 = {
+        :begin   => %r|\AContent-Type: message/delivery-status|,
+        :error   => %r/\AReason:[ \t]*(.+)\z/,
+        :rfc822  => %r|\AContent-Type: text/rfc822-headers\z|,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
+        userunknown: %r/No[ ]such[ ]user/x,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Symantec.cloud http://www.messagelabs.com'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+
+      # X-Msg-Ref: server-11.tower-143.messagelabs.com!1419367175!36473369!1
+      # X-Originating-IP: [10.245.230.38]
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->